src/main/scala/org/camunda/feel/impl/builtin/ConversionBuiltinFunctions.scala

```diff
@@ -187,6 +175,12 @@ class ConversionBuiltinFunctions(valueMapper: ValueMapper) {
   private def numberFunction3 = builtinFunction(
     params = List("from", "grouping separator", "decimal separator"),
     invoke = {
+      case List(ValString(from), ValString(grouping), ValString(decimal)) if !isNumeric(from)     =>
+        ValNull
+      case List(ValString(from), ValNull, ValString(decimal)) if !isNumeric(from)                 =>
+        ValNull
+      case List(ValString(from), ValString(grouping), ValNull) if !isNumeric(from)                =>
+        ValNull
```

Comment:
:wrench: We could combine these cases into one. 

```suggestion
      case List(ValString(from), _, _) if !isNumeric(from)     =>
        ValNull
```